pypsa/linopf.py

```diff
@@ -700,6 +700,7 @@ def map_solution(c, attr):
                     i_eff = '' if i == '1' else i
                     eff = get_as_dense(n, 'Link', f'efficiency{i_eff}', sns)
                     set_from_frame(pnl, f'p{i}', - values * eff)
+                    pnl[f'p{i}'].loc[sns, n.links.index[n.links[f'bus{i}'] == ""]] = n.component_attrs['Link'].loc[f'p{i}','default']
```

Comment:
That line is a bit long
```suggestion
                    pnl[f'p{i}'].loc[sns, n.links.index[n.links[f'bus{i}'] == ""]] = \
                                              n.component_attrs['Link'].loc[f'p{i}','default']
```